### ISSUE TITLE
Νavitation menu is jumb between pages

### ISSUE THREAD

**Issue by vbaimas** [wplemon/gridd#47]
I just realized that the navigation menu is jump between the pages. I do not know if it has anything to do with the branding area changes.

![Image of Navigation menu](https://cl.ly/095eeff4a664/download/Screen%252520Recording%2525202019-07-10%252520at%25252001.39%252520PM.gif)

**Comment by aristath** [comment]
Looks to me like the content area doesn't have enough content, so the grid tries to compensate by resizing the grid-parts.
Have you defined a height for the header? By default it's set to `auto` so it will resize to fit if the content is very short.
In the main grid you can define a height for the header by editing the `auto` value on the right of the header

**Comment by vbaimas** [comment]
I added more content in content area (About page). Also I set a height of 145px in header, but that the problem is still remaining. 
I wish I did not make any other arrangements that would affect this result. 

![Image of Navigation menu](https://cl.ly/fbffda425997/Screen%20Recording%202019-07-10%20at%2006.23%20PM.gif)

**Comment by aristath** [comment]
I assume the horizontal line is the header-shadow?
So far I've been unable to replicate this one.
The `145px` size was added to `Customizer > Theme Option > Grid`, correct? Not in `Customizer > Theme Options > Header`

**Comment by vbaimas** [comment]
Yes, the horizontal line is the header-shadow. I tried both, but the problem is still remaining.

**Comment by carolinan** [comment]
I am _sort of_ able to reproduce this on a fresh install.

Step 1: Activate Gridd Version: 1.1.16
(Step 2: For ease of testing, create a new menu with the Sample Page and assign it to Navigation 1.)
Step 3: in the Site Grid, set the header height to 145
**Test 1:**  The header size changes when I change from the homepage to sample page.
Step 4: Change 145 to 145**px** 
**Test 2:**  The header size is correct on all pages.
Step 5: Change 145px to 145**em** 
**Test 3:**  The header size is correct on all pages.
Step 6: Change 145em to 45**%** 
**Test 4:**  The header size is different on the different pages.

### Suggestion:
The details text for this option could include information about adding the unit,
and information about which unit that is assumed when no unit has been added.
